ethcore/src/block.rs

```diff
@@ -459,6 +460,32 @@ pub(crate) fn enact(
 	b.close_and_lock()
 }
 
+/// Enact the block given by `block_bytes` using `engine` on the database `db` with the given `parent` block header
+pub fn enact_verified(
```

Comment:
I don’t think this needs to be public.

Reply:
pub crate should be enough

Reply:
It is actually not used currently, it turned out to be more complicated than I thought as we need to hold a ref to a header. Will rethink the approach here.